cognite/client/_api/functions.py

```diff
@@ -1024,26 +1023,26 @@ def create(
 
         """
         _get_function_identifier(function_id, function_external_id)
-        nonce = create_session_and_return_nonce(
-            self._cognite_client, api_name="Functions API", client_credentials=client_credentials
-        )
-        body: dict[str, list[dict[str, str | int | None | dict]]] = {
-            "items": [
-                {
-                    "name": name,
-                    "description": description,
-                    "functionId": function_id,
-                    "functionExternalId": function_external_id,
-                    "cronExpression": cron_expression,
-                    "nonce": nonce,
-                }
-            ]
-        }
 
-        if data:
-            body["items"][0]["data"] = data
+        with self._cognite_client.iam.sessions(client_credentials) as session:
+            body: dict[str, list[dict[str, str | int | None | dict]]] = {
```

Comment:
This doesn't look right to me; aren't you killing the schedule right after it has been created? 🤔

Reply:
I think it is for you use the session for creating the schedule, but lets check with the functions team.